app/assets/javascripts/student_profile/ProfilePdfDialog.js

```diff
@@ -134,18 +145,22 @@ export default class ProfilePdfDialog extends React.Component {
                 datepicker: styles.datepickerContainer,
                 input: styles.datepickerInput
               }}
-              value={this.state.filterToDate.format('MM/DD/YYYY')}
+              value={this.state.filterToDateText}
               onChange={this.onFilterToDateChanged}
               datepickerOptions={{
                 showOn: 'both',
                 dateFormat: 'mm/dd/yy',
                 minDate: undefined
               }} />
           </div>
+          <div style={{height: '2em'}}>
+            {!this.areDatesValid() && <div className="PdfDialogue-warning" style={styles.invalidDate}>Choose a valid date</div>}
+          </div>
         </div>
         <div style={{display: 'flex', justifyContent: 'flex-end'}}>
           <button
-            style={styles.studentReportButton}
+            style={{background: (areDatesValid) ? undefined : '#ccc'}}
```

Comment:
This removes the `studentReportButton` styles, and the snapshot change shows that too, can you share more about why?

Reply:
This was just an oversight. I took it out to think about how to merge the styles and never added it back.